### ISSUE TITLE
What version of Pytorch do you use?

### ISSUE THREAD

**Issue by YTMartian** [GXYM/TextBPN-Plus-Plus#2]
Thanks for your job. Could you please  tell me which version of Pytorch do you use? Because some code not work in pytorch 1.11.0,thanks.

**Comment by GXYM** [comment]
> Thanks for your job. Could you please tell me which version of Pytorch do you use? Because some code not work in pytorch 1.11.0,thanks.
> 
> We test the environment “ Ubuntu20.04+Python3.8 PyTorch >= 1.10.0-cu113;”
> 
> In the this environment, you may be need to use a new parameter setting in the “data.DataLoader”，such as train_loader = data.DataLoader(trainset, batch_size=cfg.batch_size, shuffle=True, num_workers=cfg.num_workers, pin_memory=True, generator=torch.Generator(device=cfg.device))
> 
> I hope this will help you！

**Comment by GXYM** [comment]
> We test the environment “ Ubuntu20.04+Python3.8 PyTorch >= 1.10.0-cu113;”
> 
> In the this environment, you may be need to use a new parameter setting in the “data.DataLoader”，such as train_loader = data.DataLoader(trainset, batch_size=cfg.batch_size, shuffle=True, num_workers=cfg.num_workers, pin_memory=True, generator=torch.Generator(device=cfg.device))
> 
> I hope this will help you！

**Comment by YTMartian** [comment]
> > We test the environment “ Ubuntu20.04+Python3.8 PyTorch >= 1.10.0-cu113;”
> > In the this environment, you may be need to use a new parameter setting in the “data.DataLoader”，such as train_loader = data.DataLoader(trainset, batch_size=cfg.batch_size, shuffle=True, num_workers=cfg.num_workers, pin_memory=True, generator=torch.Generator(device=cfg.device))
> > I hope this will help you！

Thanks for your kindly help, it works.
